apps/predbat/web.py

```diff
@@ -430,28 +431,36 @@ async def html_log(self, request):
             start_line = line[0:27]
             rest_line = line[27:]
 
-            if "error" in line_lower:
-                text += "<tr><td>{}</td><td nowrap><font color=#ff3333>{}</font> {}</td></tr>\n".format(lineno, start_line, rest_line)
+            if (
```

Comment:
@springfall2008 this bit is where lines are filtered in/out - lines that shouldn't be seen are simply not added to the list that's passed through to the template.